presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -617,6 +617,35 @@ public static Slice rightPad(@SqlType("varchar(x)") Slice text, @SqlType(Standar
         return pad(text, targetLength, padString, text.length());
     }
 
+    @Description("computes Levenshtein Distance between two strings")
+    @ScalarFunction
+    @LiteralParameters({"x", "y"})
+    @SqlType(StandardTypes.BIGINT)
+    public static long levenshteinDist(@SqlType("varchar(x)") Slice string1, @SqlType("varchar(y)") Slice string2)
+    {
+        int[] codePoints1 = castToCodePoints(string1);
+        int[] codePoints2 = castToCodePoints(string2);
+        long[] prevDistances = new long[codePoints2.length + 1];
+        long[] currDistances = new long[codePoints2.length + 1];
+
+        for (int j = 0; j < codePoints2.length + 1; ++j) {
+            prevDistances[j] = j;
+        }
+
+        for (int i = 0; i < codePoints1.length; ++i) {
+            currDistances[0] = i + 1;
+            for (int j = 0; j < codePoints2.length; ++j) {
+                currDistances[j + 1] = codePoints1[i] == codePoints2[j] ?
```

Comment:
Usually, we don't use ternary operator since it might make the code hard to read. 

Use `if ... else ...` instead:
```
if (leftCodePoints[i] == rightCodePoints[j]) {
    distances[j + 1] = ...
}
else {
    distances[j + 1] = ...
}
```